src/components/Title.tsx

```diff
@@ -32,17 +33,22 @@ interface ITitleProps {
   titleIcon?: ReactElement<SvgProps>
   title: string
   subtitle?: string
-  description: string
+  description?: string
   children?: ReactElement
+  style?: StyleProp<ViewStyle>
 }
 
-const Title = ({ titleIcon, title, subtitle, description, children }: ITitleProps): ReactElement => (
-  <Container>
+const Title = ({ titleIcon, title, subtitle, description, children, style }: ITitleProps): ReactElement => (
+  <Container style={style}>
     {titleIcon}
     <TitleHeading>{title}</TitleHeading>
     {subtitle && <TitleSubheading>{subtitle}</TitleSubheading>}
-    <TitleContent>{description}</TitleContent>
+    {description && <TitleContent>{description}</TitleContent>}
     {children}
   </Container>
 )
 export default Title
+
+export const TitleSpacing = styled(Title)`
+  margin-bottom: ${props => props.theme.spacings.xxl};
+`
```

Comment:
Please move to the top of the file

Reply:
since i'm referring to the Title component  itself, i can't move it because i would use it before the declaration of the title component (line 41) -> rejected